package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "lol-wrapped",
+  "version": "0.0.1",
+  "private": true,
+  "workspaces": [
+    "api",
+    "ui"
+  ],
+  "main": "index.js",
+  "repository": "https://github.com/reishaleem/lol-wrapped.git",
+  "author": "Dean Haleem <deanhaleem.10@gmail.com>, Reis Haleem <reishaleem@gmail.com>",
+  "license": "MIT",
+  "scripts": {
+    "prepare": "husky install",
+    "build:ci": "lerna run build --stream",
+    "test:ci": "lerna run test --stream",
+    "format": "prettier --write '**/*.{js,ts}' '**/*.{html,md,json}' --loglevel='warn'",
+    "lint": "eslint '**/*.{js,ts}' --max-warnings 0",
+    "pipeline": "yarn lint && yarn build:ci && yarn test:ci"
+  },
+  "dependencies": {
+    "lerna": "^6.6.1"
+  },
```

Comment:
We might be able to make this a dev dependency. Thought we'd leave it as is for now and update later